### ISSUE TITLE
Clojure client

### ISSUE THREAD

**Issue by spoerri** [hoplon/castra#25]
Is it possible to use clojure for the client as well as the castra server, as opposed to clojurescript for the client?

**Comment by flyboarder** [comment]
You could use clojure to build a client java application http://nils-blum-oeste.net/desktop-guis-with-clojure-in-2017-what-are-the-options/

**Comment by spoerri** [comment]
Yes... What about castra? Can I use mkremote in one _clojure_ program to help remotely call a procedure on another clojure program?
